language-pack_de/modInfo.json

```diff
@@ -0,0 +1 @@
+{workshopId:892615445,title:Deutsche Übersetzung / German translation,image:modPreview.png}
```

Comment:
Probably shouldn't include this file in the repo. 😉

Reply:
Removed the file, thanks you Joshua ;D